src/parsers/manifest/dash/parse_periods.ts

```diff
@@ -44,12 +45,18 @@ export default function parsePeriods(
   periodsIR : IPeriodIntermediateRepresentation[],
   manifestInfos : IManifestInfos
 ): IParsedPeriod[] {
+  const periodsTimeInformations = getPeriodsTimeInformations(periodsIR, manifestInfos);
```

Comment:
Might be better to check that the length is coherent just to be sure we have a easy to debug error